src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@import 'variables';
+
+.card {
+  width: $card-width;
+  height: $card-height;
+  border: $card-border;
+  border-radius: $card-border-radius;
+  box-shadow: $card-shadow;
+  font-family: $font-family;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: $card-padding;
+  background-color: #fff;
+  transition: box-shadow 0.3s;
```

Comment:
The order of your styles is important. Here is useful link - https://9elements.com/css-rule-order/